queue/queue.py

```diff
@@ -3,10 +3,16 @@ def __init__(self):
     self.size = 0
     # what data structure should we
     # use to store queue elements?
-    self.storage = 
+    self.storage = Linked_List()
 
   def enqueue(self, item):
-    pass
+    if self.storage.head == None:
+      value = Node(item)
+      self.storage.change_head(value)
```

Comment:
The enqueue method should be adding values to the end of the queue. No action is needed on the head unless the queue is empty. Therefore, you can contain that logic of adding the value to the head and tail if the storage is empty within the change tail (should be called add to tail for clarity) method.